Description: [droidcamsrc] reduce frame available cb verbosity
 Frame available callback fires every frame. `LOG` log level suits this
 better. Helps when debugging.
Author: Ratchanan Srirattanamet <ratchanan@ubports.com>
Origin: vendor
Forwarded: not-needed
Last-Update: 2020-12-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gst/droidcamsrc/gstdroidcamsrcdev.c
+++ b/gst/droidcamsrc/gstdroidcamsrcdev.c
@@ -288,7 +288,7 @@
   gsize width, height;
   DroidMediaRect rect;
 
-  GST_DEBUG_OBJECT (src, "dev preview frame callback");
+  GST_LOG_OBJECT (src, "dev preview frame callback");
 
   pool = gst_object_ref (dev->raw_pool);
   gst_buffer_pool_acquire_buffer (pool, &buffer, NULL);
@@ -491,7 +491,7 @@
   GstBufferPool *pool;
   DroidMediaBufferInfo info;
 
-  GST_DEBUG_OBJECT (src, "frame available");
+  GST_LOG_OBJECT (src, "frame available");
 
   droid_media_buffer_get_info (buffer, &info);
 
